src/bugjira/config.py

```diff
@@ -0,0 +1,26 @@
+import json
+
+from bugjira.dao import Config
+
+
+def get_config(config_path=None, config_dict=None):
```

Comment:
why not just make this a class method on the Config object, perhaps something like Config.from_config(params)?